src/overlays/actors/ovl_En_Horse_Normal/z_en_horse_normal.c

```diff
@@ -21,45 +48,635 @@ const ActorInit En_Horse_Normal_InitVars = {
+             this->actor.posRot.pos.z == -1100.0f) ||
+            (this->actor.posRot.pos.x == 880.0f && this->actor.posRot.pos.y == 0.0f &&
+             this->actor.posRot.pos.z == -1170.0f)) {
+            func_80A6C6B0(this);
+            return;
+        }
+    } else if (globalCtx->sceneNum == SCENE_MALON_STABLE) {
+        if (!gSaveContext.nightFlag) {
+            Actor_Kill(&this->actor);
+        } else {
+            func_800A663C(globalCtx, &this->skin, &D_06009FAC, &D_06004580);
+            SkelAnime_ChangeAnimDefaultStop(&this->skin.skelAnime, sAnimations[this->animationIdx]);
+            func_80A6C6B0(this);
+        }
+        return;
+    } else if (globalCtx->sceneNum == SCENE_SPOT12) {
+        if (this->actor.posRot.pos.x == 3707.0f && this->actor.posRot.pos.y == 1413.0f &&
+            this->actor.posRot.pos.z == -665.0f) {
+            func_800A663C(globalCtx, &this->skin, &D_06009FAC, &D_06004580);
+            SkelAnime_ChangeAnimDefaultStop(&this->skin.skelAnime, sAnimations[this->animationIdx]);
+            func_80A6C4CC(this);
+            return;
+        }
+        func_800A663C(globalCtx, &this->skin, &D_06009FAC, &D_06004580);
+        SkelAnime_ChangeAnimDefaultStop(&this->skin.skelAnime, sAnimations[this->animationIdx]);
+    } else {
+        func_800A663C(globalCtx, &this->skin, &D_06009FAC, &D_06004580);
+        SkelAnime_ChangeAnimDefaultStop(&this->skin.skelAnime, sAnimations[this->animationIdx]);
+    }
+    if ((this->actor.params & 0xF0) == 0x10 && (this->actor.params & 0xF) != 0xF) {
+        func_80A6B91C(this, globalCtx);
+    } else {
+        func_80A6BC48(this);
+    }
+}
+
+void EnHorseNormal_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnHorseNormal* this = THIS;
+
+    func_800A6888(globalCtx, &this->skin);
+    Collider_DestroyCylinder(globalCtx, &this->bodyCollider);
+    Collider_DestroyCylinder(globalCtx, &this->cloneCollider);
+    Collider_DestroyJntSph(globalCtx, &this->headCollider);
+}
+
+void func_80A6B91C(EnHorseNormal* this, GlobalContext* globalCtx) {
+    this->actor.flags |= 0x10;
+    this->type = 4;
+    this->animationIdx = 6;
+    this->waypoint = 0;
+    this->actor.speedXZ = 7.0f;
+    SkelAnime_ChangeAnim(&this->skin.skelAnime, sAnimations[this->animationIdx], func_80A6B30C(this), 0.0f,
+                         SkelAnime_GetFrameCount(&sAnimations[this->animationIdx]->genericHeader), 2, 0.0f);
+}
+
+void EnHorseNormal_FollowPath(EnHorseNormal* this, GlobalContext* globalCtx) {
+    Path* path = &globalCtx->setupPathList[this->actor.params & 0xF];
+    Vec3s* pointPos = SEGMENTED_TO_VIRTUAL(path->points);
+    f32 dx;
+    f32 dz;
+    s32 pad;
+
+    pointPos += this->waypoint;
+    dx = pointPos->x - this->actor.posRot.pos.x;
+    dz = pointPos->z - this->actor.posRot.pos.z;
+    Math_SmoothScaleMaxMinS(&this->actor.posRot.rot.y, Math_atan2f(dx, dz) * 10430.378f, 10, 2000, 1);
```

Comment:
```suggestion
    Math_SmoothScaleMaxMinS(&this->actor.posRot.rot.y, Math_atan2f(dx, dz) * (0x8000 / M_PI), 10, 2000, 1);
```
`0x8000` may need to be `(f32) 0x8000` for matching (`10430.378f` is also somewhere else in the file)

also, use hexadecimal for 10/2000, we use hex for rotations (and yeah they sometimes didnt)

Reply:
I went ahead and used hex for the angles but unfortunately neither `(0x80000 / M_PI)` or `((f32)0x80000 / M_PI)` match.